### PR TITLE
Dont defer do dofree with batching

### LINKED ISSUE #79: Do/Dofree/Dor commands delayed after addbalanceful with batching
When using batching, the sending of dofree/do/dor (via expandAlias) is delayed until after the current batch, See https://github.com/svof/svof/blob/master/raw-svo.skeleton.lua#L741

If then things are added to the batch via sendc in svo.balanceful() to use up balance and thus cancel out dofree among other things.

### PR DESCRIPTION
**Pull request by keneanung**
This will fix #79 when merged. While usages of sendc with a table will break the current batch up, this makes sure we can run things we have no control over the way send is called in the queue (as it's done with expandAlias).

I was also stupid and included the commit of #111 here, so the merge should happen afterwards.

### CODE REVIEW

**Comment by vadimnehta** [comment]
You can submit a clean PR by making another branch and cherry-picking the commit into it.

File: raw-svo.skeleton.lua [hunk 1]
```diff
@@ -733,15 +733,14 @@ make_gnomes_work_async = function()
   signals.sysdatasendrequest:block(cnrl.processusercommand)
 
   if conf.commandecho and (conf.commandechotype == "fancy" or conf.commandechotype == "fancynewline") then
+    oldsend = send
     send = fancysend
 
     -- delay expandAlias execution, used in dor, to after batch completes
```
Comment (vadimnehta) [inline raw-svo.skeleton.lua]: I would say the comment needs to be updated?

**Comment by keneanung** [comment]
There we go. I used `git reset --hard` to reset to the commit before the sylvan changes and then cherry-picked the commit by sha and force pushed it.

**Comment by vadi2** [comment]
+1 if you've tested this lots

**Pull request merged**